client/src/components/Instructions/index.js

```diff
@@ -1,103 +1,102 @@
+            Let's compile the Counter contract, follow the <span className={styles.inline}>cli</span>  prompts.
           </div>
           <div className={styles.code}>
-            <code>
-              zos push --deploy-dependencies
-            </code>
+            <code>zos push</code>
           </div>
         </div>
         <div className={styles.step}>
           <div className={styles.instruction}>
-            We create an instance of our contract and deploy it.
+            We create an instance of our contract and deploy it, follow the <span className={styles.inline}>cli</span> 
+            prompts.
           </div>
           <div className={styles.code}>
-            <code>
-              zos create Counter --init initialize --args 2
-            </code>
+            <code>zos create Counter</code>
           </div>
         </div>
         <div className={styles.step}>
           <div className={styles.instruction}>
-            Add a log to verify that the contract is available. Before line 77 in App.js, add:
+            Add a log to verify that the contract is available. Before line 77
+            in App.js, add:
           </div>
           <div className={styles.code}>
-            <code>
-              console.log(instance);
-            </code>
+            <code>console.log(instance);</code>
           </div>
-          <p> You should see a log that contains the Counter instance in your browser console </p>
+          <p>
+            {" "}
+            You should see a log that contains the Counter instance in your
+            browser console{" "}
+          </p>
         </div>
         <div className={styles.step}>
           <div className={styles.instruction}>
-            Done! Refresh the page to interact with your instance of the counter contract.
+            Done! Refresh the page to interact with your instance of the counter
+            contract.
+          </div>
+          <div className={styles.instruction}>
+            Learn more about <a href="https://zeppelinos.org/">ZeppelinOS</a> or
+            ask a question in the{" "}
+            <a href="https://forum.zeppelin.solutions">Forum</a>.
           </div>
-          <Button onClick={() => window.location.reload()}>
-            Reload
-          </Button>
+          <Button onClick={() => window.location.reload()}>Reload</Button>
         </div>
       </div>
     );
   }
 
   renderSetup() {
-    const isProd = process.env.NODE_ENV === 'production';
+    const isProd = process.env.NODE_ENV === "production";
     if (!isProd) {
       return (
         <div className={styles.instructions}>
           <h1> ZepKit is running! </h1>
           <div className={styles.step}>
             <div className={styles.instruction}>
-              Congratulations! Your application is correctly setup.
-              Now, you have two options:
+              Congratulations! Your application is correctly setup. Now, you
```

Comment:
This is no longer true. `npm run start-blank` doesn't exist anymore.

Reply:
Good to know!